scripts/verify.bash

```diff
@@ -13,7 +13,7 @@ VERSION=`go version | awk '{print $3}'`
 echo "go version $VERSION"
 
 echo "checking: go fmt ..."
-BADFMT=`find * -name '*.go' -not -name '.#*' | xargs gofmt -l`
+BADFMT=`find * -name '*.go' -not -name '.#*' | xargs gofmt -l -s`
 if [ -n "$BADFMT" ]; then
     BADFMT=`echo "$BADFMT" | sed "s/^/  /"`
     echo -e "gofmt is sad:\n\n$BADFMT"
```

Comment:
Can we add the gofmt command run here, so folks know what to look for?

Reply:
Like this ?

```sh
go version go1.10.1
checking: go fmt ...
gofmt is sad:

gofmt -l -s -w wrench/wrench.go
gofmt -l -s -w wrench/wrench_test.go
```